edk2toolext/nuget_publishing.py

```diff
@@ -368,6 +404,14 @@ def GatherArguments():
                             required=True)
         parser.add_argument('--Author', dest="Author", help="<Required> Author string for publishing", required=True)
         parser.add_argument("--ProjectUrl", dest="Project", help="<Required> Project Url", required=True)
+        parser.add_argument("--RepositoryType", dest="RepositoryType", help="<Optional> Repository Type",
```

Comment:
I would add these to an argument group. `g = parser.add_argument_group(title = "", description="")`. Then `g.add_argument()`. It doesn't do anything extra logically but will help the user out a bit when doing `-h, --help`

Reply:
Done in PR commit https://github.com/tianocore/edk2-pytool-extensions/pull/429/commits/2682256cc76730f18db5526ac045b621f119f4db